gallery/src/styles/NxFormLayout/NxFormHorizontalLayoutExample.tsx

```diff
@@ -79,7 +79,7 @@ export default function NxFormLayoutExample() {
         </NxRadio>
       </fieldset>
       <footer className="nx-form-footer">
-        <NxErrorAlert>This is an example <strong>error</strong> message.</NxErrorAlert>
+        <NxInfoAlert className="nx-alert--modifier">This is a sample alert message</NxInfoAlert>
```

Comment:
I don't think this is doing anything.
```suggestion
        <NxInfoAlert>This is a sample alert message</NxInfoAlert>
```